tools/check_format.py

```diff
@@ -86,15 +86,19 @@ def isBuildFile(file_path):
 
 
 def checkFileContents(file_path):
-  message = "%s has over-enthusiastic spaces:" % file_path
-  findSubstringAndPrintError('.  ', file_path, message)
+  findSubstringAndPrintError('.  ', file_path, "over-enthusiastic spaces")
+  findSubstringAndPrintError('#include <envoy', file_path,
```

Comment:
Doesn't this basically apply to all project includes? Not just "envoy" ones?

Reply:
Yeah, we want `<test` and `<common` as well I think.

Reply:
And extensions now. Can we potentially somehow share the list with the header order fixer?

Reply:
(common py include potentially?)

Reply:
Good call.  in header_order.py there's a similar list, expressed as:
```
      file_header_filter(),
      regex_filter('<.*\.h>'),
      regex_filter('<.*>'),
      regex_filter('"envoy/.*"'),
      regex_filter('"common/.*"'),
      regex_filter('"source/common/.*"'),
      regex_filter('"exe/.*"'),
      regex_filter('"server/.*"'),
      regex_filter('"extensions/.*"'),
      regex_filter('"test/.*"'),
```

The 7 of these could be generated from a shared array of strings.  Would it be cool to effectively change "/source/common/.*" above to just "/source/.*"?  Specifically, if someone loads from source/server what should happen?   It seems like that maybe the intent is that it shouldn't pass lint, and it should not be auto-fixable either.  Or we could -- at the tooling level -- just ignore that.

Reply:
Yeah I think that would be fine.

Reply:
done